engine/src/main/scala/cromwell/engine/workflow/lifecycle/MaterializeWorkflowDescriptorActor.scala

```diff
@@ -335,11 +335,11 @@ class MaterializeWorkflowDescriptorActor(serviceRegistryActor: ActorRef,
   }
 
   private def validateNamespaceWithImports(w: WorkflowSourceFilesWithDependenciesZip): ErrorOr[WdlNamespaceWithWorkflow] = {
-    def getMetadatae(importsDir: Path, prefix: String = ""): Seq[(String, Path)] = {
-      importsDir.children.toSeq flatMap {
+    def getMetadatae(importsDir: Path, prefix: String = ""): List[(String, Path)] = {
```

Comment:
Since this ultimately winds up being used in the `WriteMetadataActor` which is converting things to a `Vector`, we can save a `toList` and then a `toVector` by just calling `toVector` here and returning `Vector`